### PR TITLE
Add a Gitter chat badge to README.md

### PR DESCRIPTION
**Pull request by gitter-badger**
### Deducteam/lambdapi now has a Chat Room on Gitter

@gabrielhdt has just created a chat room. You can visit it here: [https://gitter.im/dedukti3/community](https://gitter.im/dedukti3/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&content=body_link).

This pull-request adds this badge to your README.md:

[Gitter](https://gitter.im/dedukti3/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge)

If my aim is a little off, please [let me know](https://gitlab.com/gitlab-org/gitter/readme-badger/issues).

Happy chatting.

PS: [Click here](https://gitter.im/settings/badger/opt-out) if you would prefer not to receive automatic pull-requests from Gitter in future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,82 @@
-Lambdapi- Implementation of the λΠ-calculus modulo rewriting
-============================================================
+Lambdapi, a proof assistant based on the λΠ-calculus modulo rewriting
+=====================================================================
 
-Lambdapi is an implementation of the λΠ-calculus modulo rewriting, which
-is the system of Dedukti (https://github.com/Deducteam/Dedukti).
+[![Join the chat at https://gitter.im/dedukti3/community](https://badges.gitter.im/dedukti3/community.svg)](https://gitter.im/dedukti3/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (fblanqui) [inline README.md:4]: Perhaps it needs to be updated now?

Comment (gabrielhdt) [reply README.md:4]: It may be better to do the change ourselves, the diff is weird. The url for the badge is accessible from the room

Comment (fblanqui) [reply README.md:4]: Do you mean that we should close this PR and add the badge in the README ourself?

Comment (gabrielhdt) [reply README.md:4]: Yeah I think so.

**Pull request closed without merging**